addons/pos_hr/static/src/js/PaymentScreen.js

```diff
@@ -0,0 +1,18 @@
+odoo.define('pos_hr.PaymentScreen', function (require) {
+    'use strict';
+
+    const PaymentScreen = require('point_of_sale.PaymentScreen');
+    const Registries = require('point_of_sale.Registries');
+
+    const PosHrPaymentScreen = (PaymentScreen_) =>
```

Comment:
typo here
```suggestion
    const PosHrPaymentScreen = (PaymentScreen) =>
```

Reply:
It's not a typo. I use another name to don't confuse with variable defined earlier

```js
    const PaymentScreen = require('point_of_sale.PaymentScreen');
```